.zenodo.json

```diff
@@ -1,44 +1,67 @@
 {
-    "description": "<p>The signac-flow tool provides the basic components to setup simple to complex workflows for signac projects (https://glotzerlab.engin.umich.edu/signac). That includes the definition of data pipelines, execution of data space operations and the submission of operations to high-performance super computers.</p>",
-    "license": "BSD-3-Clause",
-    "upload_type": "software",
-    "creators": [
+    "contributors": [
         {
             "affiliation": "University of Michigan",
-            "name": "Carl S. Adorf"
+            "name": "Benjamin Swerdlow",
+            "orcid": "0000-0001-6240-1430"
         },
         {
             "affiliation": "University of Michigan",
-            "name": "Paul M. Dodd"
+            "name": "Joshua A. Anderson"
         },
         {
             "affiliation": "University of Michigan",
-            "name": "Vyas Ramasubramani"
+            "name": "Jens Glaser",
+            "orcid": "0000-0003-1852-3849"
         },
         {
             "affiliation": "University of Michigan",
-            "name": "Bradley Dice"
+            "name": "Timothy Moore",
+            "orcid": "0000-0002-5709-7259"
+        }
+    ],
+    "creators": [
+        {
+            "affiliation": "University of Michigan",
+            "name": "Carl Simon Adorf",
+            "orcid": "0000-0003-4962-2495"
         },
         {
             "affiliation": "University of Michigan",
-            "name": "Benjamin Swerdlow"
+            "name": "Vyas Ramasubramani",
+            "orcid": "0000-0001-5181-9532"
         },
         {
             "affiliation": "University of Michigan",
-            "name": "Joshua Anderson"
-        }
-    ],
-    "access_right": "open",
-    "related_identifiers": [
+            "name": "Bradley D. Dice",
+            "orcid": "0000-0002-9983-0770"
+        },
+        {
+            "affiliation": "Boise State University",
+            "name": "Mike Henry",
+            "orcid": "0000-0002-3870-9993"
+        },
         {
-            "scheme": "url",
-            "identifier": "https://dx.doi.org/10.1016/j.commatsci.2018.01.035",
-            "relation": "isSupplementedBy"
+            "affiliation": "University of Michigan",
+            "name": "Paul M. Dodd"
         },
         {
-            "scheme": "url",
-            "identifier": "arXiv://1611.03543 [cs.DB]",
-            "relation": "isSupplementedBy"
+            "affiliation": "University of Michigan",
+            "name": "Sharon C. Glotzer",
+            "orcid": "0000-0002-7197-0085"
         }
-    ]
-}
+    ],
+    "description": "The signac-flow tool provides the basic components to set up simple to complex workflows for projects as part of the signac framework. That includes the definition of data pipelines, execution of data space operations and the submission of operations to high-performance super computers.",
+    "keywords": [
+        "python",
+        "reproducibility",
+        "sharability",
```

Comment:
typo: shareability, but I also don't think that shareability applies to signac-flow as much as signac. I would consider "automation" or "high performance computing" or similar.

Reply:
Thank's for the catch! Are you open to the argument that signac-flow enables sharing of workflows? I imagine that down the road sharing of project-implementations becomes more common.